File: 1Latihan_B/Java/DaftarTimSepakbola.java
public class DaftarTimSepakbola {
    private String nama;
    private String negara;
    private String tahun_berdiri;
    private Pemain posisi;

    public DaftarTimSepakbola() {
        posisi = new Pemain();
    }

    Pemain getPosisi() {
        return posisi;
    }

    class Pemain {
        Pemain() {
        }

        void kiper(String nama) {
            System.out.println(nama);
        }

        void bek(String nama) {
            System.out.println(nama);
        }

        void gelandang(String nama) {
            System.out.println(nama);
        }

        void penyerang(String nama) {
            System.out.println(nama);
        }
    }

    public DaftarTimSepakbola(String nama, String negara, String tahun_berdiri/* , String pemain */) {
        this.nama = nama;
        this.negara = negara;
        this.tahun_berdiri = tahun_berdiri;
    }

    public void setNama(String nama) {
        this.nama = nama;
    }

    public String getNama() {
        return this.nama;
    }

    public void setNegara(String negara) {
        this.negara = negara;
    }

    public String getNegara() {
        return this.negara;
    }

    public void setTahunBerdiri(String tahun_berdiri) {
        this.tahun_berdiri = tahun_berdiri;
    }

    public String getTahunBerdiri() {
        return this.tahun_berdiri;
    }

    public void printhasil() {
        System.out.println("Nama : " + this.nama);
        System.out.println("Negara : " + this.negara);
        System.out.println("Tahun Berdiri : " + this.tahun_berdiri);
    }
}

File: 1Latihan_B/Python/DaftarTimSepakBola.py
class DaftarTimSepakBola():
    __nama = ""
    __negara = ""
    __tahun_berdiri = ""
    __pemain = ""

    def __init__(self, nama="", negara="", tahun_berdiri="", pemain=""):
        self.__nama = nama
        self.__negara = negara
        self.__tahun_berdiri = tahun_berdiri
        self.__pemain = pemain

    def setNama(self, nama):
        self.__nama = nama

    def getNama(self):
        return self.__nama

    def setNegara(self, negara):
        self.__negara = negara

    def getNegara(self):
        return self.__negara

    def setTahunBerdiri(self, tahun_berdiri):
        self.__tahun_berdiri = tahun_berdiri

    def getTahunBerdiri(self):
        return self.__tahun_berdiri

    def setPemain(self, pemain):
        self.__pemain = pemain

    def getPemain(self):
        return self.__pemain

    def printHasil(self):
        print("Nama:" + str(self.__nama))
        print("Negara:" + str(self.__negara))
        print("Tahun Berdiri:" + str(self.__tahun_berdiri))
        print("Pemain:" + str(self.__pemain))
